tools/python/onnx_test_data_utils.py

```diff
@@ -210,6 +220,21 @@ def get_arg_parser():
 
         data = create_random_data(args.shape, args.datatype, args.min_value, args.max_value, args.seed)
         numpy_to_pb(args.name, data, args.output)
+    elif args.action == "raw_to_pb":
+        if not args.output or not args.name:
+            print("Missing argument. Need output and name to be specified.", file=sys.stderr)
+            sys.exit(-1)
+
+        data = np.fromfile(args.input, dtype=np.ubyte)
+        numpy_to_pb(args.name, data, args.output)
+    elif args.action == "string_to_pb":
+        if not args.output or not args.name:
+            print("Missing argument. Need output and name to be specified.", file=sys.stderr)
+            sys.exit(-1)
+
+        data = np.ndarray((1), dtype=object)
+        data[0] = args.input
+        numpy_to_pb(args.name, data, args.output)
     elif args.action == "update_name_in_pb":
         if not args.input or not args.name:
```

Comment:
not related to this PR, but it looks like this should also check for `args.output`

Reply:
that one writes to input file if output isn't specified. the description of the helper mentions this.

Reply:
ah, I see. never mind that then :)